datasets/load_datasets.py

```diff
@@ -41,7 +40,7 @@ def try_load_dataset(dataset_name, output_directory):
     if dataset_name in dataset_loaders.keys():
         try:
             return dataset_loaders[dataset_name](output_directory)
-        except:
+        except BaseException:
             logging.warning("Internal error loading dataset")

```

Comment:
In this handling `except BaseException` is not better than `except`. Add traceback in warning or print exception name at least:
```
except BaseException as e:
    logging.warning(f"Internal error loading dataset: {e}")
```

Reply:
done